string name
float32 distance
float32 elevation